### PR TITLE
Fix phone auth api login

### LINKED ISSUE #87: Sms is depreciated
The entire sms portion is depreciated. I think that they are missing required headers to preform any actions.

### PR DESCRIPTION
**Pull request by aarlin**
Tinder moved away from account kit login - being deprecated later next year I believe

Fixed the README and phone_auth_token.py for these new changes

Fixes #87 and #89

### CODE REVIEW

**Review by fbessez — APPROVED** [review]
LGTM

**Comment by aarlin** [comment]
Did you want to merge this change? I don't have write access to merge

File: README.md [hunk 1]
```diff
@@ -62,10 +62,22 @@ Note: All curls must be sent with the headers as well (the only exception is tha
...
-		 <td>For SMS authentication (two-factor)</td>
-		 <td>{'token': INSERT_HERE, 'id': INSERT_HERE, 'client_version':'9.0.1'}</td>
+      <tr>
+       <td>/v2/auth/sms/send?auth_type=sms</td>
+       <td>Part 1 of SMS authentication (two-factor)</td>
+       <td>{'phone_number': INSERT_HERE}</td>
+		 <td>POST</td>
+	  </tr>
+      <tr>
+		 <td>/v2/auth/login/sms</td>
```
Comment (gomisha) [inline README.md]: part 2 is /v2/auth/sms/validate?auth_type=sms

**Review by gomisha — APPROVED** [review]
tested - works - token generated

File: README.md (hunk 1 above)
Comment (aarlin) [reply README.md]: Good catch, updated

**Review by gomisha — APPROVED** [review]

**Comment by Sammeeey** [comment]
@aarlin 
When I work on your fork I have the following issue:

I open the jupyter notebook and add
```
import phone_auth_token
```
in the first line, so that I have the following:
```
import tinder_api
import fb_auth_token
import phone_auth_token
```

When I type in my phone number like 0049152[...] I get the following error:
```
C:\Users\Name\Anaconda3\lib\site-packages\urllib3\connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)

https://api.gotinder.com/v2/auth/sms/send?auth_type=sms

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-4f4ef5614574> in <module>
----> 1 import phone_auth_token

~\PycharmProjects\Tinder2020\Tinder\phone_auth_token.py in <module>
     36 
     37 phone_number = input("Please enter your phone number under the international format (country code + number): ")
---> 38 log_code = send_otp_code(phone_number)
     39 otp_code = input("Please enter the code you've received by sms: ")
     40 refresh_token = get_refresh_token(otp_code, phone_number)

~\PycharmProjects\Tinder2020\Tinder\phone_auth_token.py in send_otp_code(phone_number)
     13     print(r.url)
     14     response = r.json()
---> 15     if(response.get("data")['sms_sent'] == False):
     16         return False
     17     else:

TypeError: 'NoneType' object is not subscriptable


```

I think the API is not even accessible or what is happening?

**Comment by gelodefaultbrain** [comment]
> @aarlin When I work on your fork I have the following issue:
> 
> I open the jupyter notebook and add
> 
> ```
> import phone_auth_token
> ```
> 
> in the first line, so that I have the following:
> 
> ```
> import tinder_api
> import fb_auth_token
> import phone_auth_token
> ```
> 
> When I type in my phone number like 0049152[...] I get the following error:
> 
> ```
> C:\Users\Name\Anaconda3\lib\site-packages\urllib3\connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
>   InsecureRequestWarning)
> 
> https://api.gotinder.com/v2/auth/sms/send?auth_type=sms
> 
> ---------------------------------------------------------------------------
> TypeError                                 Traceback (most recent call last)
> <ipython-input-4-4f4ef5614574> in <module>
> ----> 1 import phone_auth_token
> 
> ~\PycharmProjects\Tinder2020\Tinder\phone_auth_token.py in <module>
>      36 
>      37 phone_number = input("Please enter your phone number under the international format (country code + number): ")
> ---> 38 log_code = send_otp_code(phone_number)
>      39 otp_code = input("Please enter the code you've received by sms: ")
>      40 refresh_token = get_refresh_token(otp_code, phone_number)
> 
> ~\PycharmProjects\Tinder2020\Tinder\phone_auth_token.py in send_otp_code(phone_number)
>      13     print(r.url)
>      14     response = r.json()
> ---> 15     if(response.get("data")['sms_sent'] == False):
>      16         return False
>      17     else:
> 
> TypeError: 'NoneType' object is not subscriptable
> ```
> 
> I think the API is not even accessible or what is happening?

as of now it's same thing with me, not working anymore. I can't even receive the OTP

**Pull request merged**